### DISCUSSION TITLE
How to connect RabbitMQ which is installed on other server (PC)

### DISCUSSION THREAD

**Discussion by nitin-kumar-mgr** [rabbitmq/rabbitmq-server#12207] [Questions] [answered]
I want to connect RabbitMQ server which is installed on our other host machine [IP - 192.168.0.115]. and it runs locally on "localhost:15672" and i am trying to connect using my C# code.

This is my code to connect RabbitMQ from my local machine which is also connected in a same intranet environment.

var factory = new ConnectionFactory
                {
                    HostName = "192.168.0.115",
                    UserName = "guest",         
                    Password = "guest",        
                    Port = 15672,                     
                                                       
                };
                I want to connect RabbitMQ server which is installed on our other host machine [IP - 192.168.0.115]. and it runs locally on "localhost:15672" and i am trying to connect using my C# code.

This is my code to connect RabbitMQ from my local machine which is also connected in a same intranet environment.

               ```
 var factory = new ConnectionFactory
                {
                    HostName = "192.168.0.115",
                    UserName = "guest",         
                    Password = "guest",        
                    Port = 15672,                     
                                                       
                };
```
by using this code i am getting an exception of "RabbitMQ.Client.Exceptions.BrokerUnreachableException: 'None of the specified endpoints were reachable'".

**Comment by mkuratczyk** [comment, marked as answer]
15672 is the Management UI/API port. For AMQP use 5672

**Reply by michaelklishin** [reply]
And the list of [other ports used by RabbitMQ](https://www.rabbitmq.com/docs/networking#ports) has a dedicated doc section.

Plus the sections on [default user's connectivity limitations](https://www.rabbitmq.com/docs/access-control#loopback-users) and how to narrow down [connectivty issues more efficiently](https://www.rabbitmq.com/docs/troubleshooting-networking) (without guessing).
